cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from .models import Movie
+
+class MovieSerializer(serializers.Serializer):

```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for simpler and more efficient code when dealing with Django models. This will automatically generate the `create` and `update` methods and handle field definitions based on the model.